packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -76,6 +78,13 @@ function VisCanvas(props: PropsWithChildren<Props>) {
             <ThresholdAdjuster value={raycasterThreshold} />
           )}
         </VisCanvasProvider>
+
+        <Html>
+          <svg
+            ref={(elem) => setSvgOverlay(elem || undefined)}
+            className={styles.svgOverlay}
+          />
+        </Html>
```

Comment:
Don't you need to set the size to the `canvasSize` so that svg elements are properly positioned ?

Reply:
The size is set in CSS to 100%, so I think it's fine.